### PR TITLE
Flight model update and full landing flaps to 40

### LINKED ISSUE #2247: [BUG] Cannot remain on runway in high crosswinds - both takeoff and landing
**Mod Version**
master

"built": "2020-12-05T18:24:08+00:00",
  "ref": "",
  "sha": "",
  "actor": "Chris Johnson <cj@hyperfish.com>",
  "event_name": "manual"

**Describe the bug**

Under high crosswind conditions, the airplane will weathervane and there is not enough rudder authority to stay on the runway.

**To Reproduce**

1.Insert a 90-degree crosswind of 38 knots (with or without gusts, but they need to be set to the same direction)
2.Attempt a takeoff - the lighter the weight, the more difficult it is
3.

**Expected behavior**

Although it is not meant to be an easy to accomplish maneuver, I expected to be able to maintain something fairly close to the runway centerline (or at least remain on the runway) and complete a successful takeoff.  Landing is just as challenging.

**Actual behavior**

The airplane drifted off the side of the runway

**References**

**Additional context**
I will be submitting a pull request for this. Per the instructions for submitting pull requests, I am opening this issue for it first.
Also, although this may be related to https://github.com/flybywiresim/a32nx/issues/1687, the originator of that issue said the issue he was reporting was independent of wind.

Was this working before/when did the issue start occurring?No

Is this a problem in the vanilla unmodded game?Yes

Discord username (if different from GitHub):donbikes#4084

Screenshot (359)
(Note: 38 knots crosswind is the maximum demonstrated value for the airplane's capability, the 35 knot limitation is due to an engine issue in crosswinds. The airplane in the sim should be able to demonstrate a takeoff in 38 knot crosswinds even though in operations the engine limitation will not allow it).

### LINKED ISSUE #2248: [BUG] Wrong Flap Deflection Angle for Full Landing Flaps
**Mod Version**
master

built": "2020-12-05T18:24:08+00:00",
"ref": "",
"sha": "",
"actor": "Chris Johnson cj@hyperfish.com",
"event_name": "manual"

**Describe the bug**

Asobo used the A320 CEO flap deflection angle of 35 degrees instead of the 40 degrees used in the NEO. This leads to an incorrect animation for the exterior view of the airplane when full flaps are deployed and the flap configuration symbol in the ECAM is wrong.

**To Reproduce**

1.Open the flight_model.cfg file and note the following line in the section `{FLAPS.0}` flaps-position.4 = 35, 177, 1.93
2.The number "35" should be "40" for the NEO
3.Observe the ECAM symbol for the configuration when full landing flaps are deployed

**Expected behavior**

Screenshot (362)

**Actual behavior**

**References**

**Additional context**
I will be submitting a pull request for this. Per the instructions for submitting pull requests, I am opening this issue for it first. @MisterChocker will be supplying the change for the ECAM symbology.

Was this working before/when did the issue start occurring? No

Is this a problem in the vanilla unmodded game? Yes

Discord username (if different from GitHub):donbikes#4084

### PR DESCRIPTION
**Pull request by donstim**
Fixes #2247  and Fixes #2248 

## Summary of Changes
      1. Changes rudder geometry and scalars in the flight model.cfg file to provide more realistic crosswind takeoff and landing capability.
       2. Changes the values of flight model variables used only in the legacy flight model for consistency with the modern flight model.
       3. Changes the flap deflection for CONF FULL from 35 degrees to 40 degrees in the flight model, which also changes the deflection shown in the external view.
       4. Changes the flap configuration symbol in the ECAM to show a 40 degree flap deflection for CONF FULL (by @MisterChocker ).

## References
 Screenshot (359)
Screenshot (362)
 

## Additional context

 1. For testing purposes:  While the maximum demonstrated crosswind capability is 38 knots for both takeoff and landing, there is an engine limitation that limits takeoffs to 35 knots crosswind. Since this engine behavior is not simulated, for testing purposes of this PR, use 38 knots crosswind for both takeoff and landing.
        2. Taking off and landing in high crosswinds is not a simple task. Unless you are a RL A320 pilot, I recommend working up to the maximum crosswind value gradually and incrementally. Practice, practice, practice!  The most critical condition is light weight. I used 110,000 lb (50 T) in developing the flight model changes.
        3. For takeoff, the first task will be to show that you can keep the airplane on the runway (hopefully with some margin to spare)! Be aware that the rudder is not very effective at low speeds, becoming more and more effective as speed increases. Be careful not to over or under control and get into a PIO on the runway! The second task will be to transition to straight flight without too much rockin' and rollin'.  You will need to use some aileron control to maintain a wing level attitude.
         4. For landing, the main task will be to decrab to align with the runway, and then to keep the airplane close to centerline on rollout. From the Flightcrew Training Manual:

Screenshot (364)
        5. I am a little worried that each person's results may be significantly affected by the rudder control being used (i.e. pedals, twist controls, etc.) and their sensitivity settings and dead zones. I used CH Products Pro Pedals to develop the values used in the flight model in this PR.
        6. Also, keep in mind that the rudder for this airplane is only intended to be used (in normal situations) for crosswind takeoff and landings, engine failures, and other large thrust asymmetries. Rudder for turn coordination is automatically applied with no pilot input needed or desired.
         7. The changes made to many of the parameters in the flight_model.cfg file affect only the legacy flight model and are intended to bring it closer to the modern flight model. It is my understanding that the autopilot uses the legacy flight model, I hope that this may improve autopilot flight dynamics (and, conversely, does not make it worse in any discernible way).
 
Discord username (if different from GitHub):donbikes#4084

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by donstim** [comment]
Alright, I see the hour or more I spent filling in the template section for summary of changes, reference documents, and additional content just disappeared into thin air. So, I will try to recreate it here.

Summary of Changes:
1. Changes the vertical tail area and span to match the real airplane and the rudder effectiveness and max angle scalar, all in the flight_model.cfg file,  to provide more realistic crosswind takeoff and landing capability.
2. Changes the legacy flight model parameters in the flight_model.cfg file to be consistent with the modern flight model.
3. Changes the CONF FULL flap deflection angle from 35 degrees to 40 degrees.
4. Changes the ECAM symbol for the CONF FULL configuration to show a flap deflection of 40 degrees (to be added by @MisterChocker )

References:
Screenshot (359)
Screenshot (362)

Additional Context:
1. Although there is an engine issue that limits the NEO crosswind takeoff capability to 35 knots, for the purpose of testing this PR, use a 38 knot crosswind for both takeoff and landing. (The engine issue isn't simulated, and we are interested in the airplane's actual capability.) Gusts are included in that 38 knots, so you can turn the gusts off, but I used the default gust value in the same direction as the crosswind when developing the flight model changes.

2. Taking off and landing in high crosswinds is not a simple task. I recommend starting with lighter crosswinds and gradually working your way up to the maximum crosswind value. Practice, practice, practice! Light weights are most critical - I used 110,000 lb (50 T) in developing the flight model changes.

3. For takeoff, the first task is to stay on the runway, and if not close to the centerline, at least away from the edges. The rudder is not very effective at low speeds, and its effectiveness increases with speed. Try not to under control at first and over control later -- it's easy to get into a PIO if you're not careful. The second task is to transition somewhat gracefully into flight. Try to keep your wings leve with small aileron inputs as you near rotation.

4. For landing, the tasks are to decrab and align with the runway just before landing, and then to stay near the centerline on rollout. From the A320neo FlightCrew Training Manual:

Screenshot (364)

5. Keep in mind that the rudder should only be used (in normal situations) for crosswind takeoff and landings, engine failures, and other large thrust asymmetries. Rudder is automatically applied for turn coordination.

6. The results these changes supply are likely to be affected by the type and model of rudder control being used (e.g., rudder pedals vs twist type joysticks, etc.) and the sensitivity settings and dead zones for the particular controller. I used CH Products Pro Rudder Pedals in developing these changes.

7.  It is my understanding that the autopilot uses the legacy flight model parameters. That is one reason for the changes to make these parameters match the modern flight model. Hopefully, the changes will improve, or at least not worsen in any way, the flight dynamics of the autopilot.

**Comment by beheh** [comment]
By changing your description from `Fixes #2247 and #2248` to `Fixes #2247 and fixes #2248` you can ensure that both issues are closed when this is merged (otherwise only the first is closed).

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -138,6 +138,9 @@
 1. [ECAM] Add icing-related ECAM warnings and memos - @beheh (Benedict Etzel)
 1. [CDU] Fix CLR position in scratchpad - @beheh (Benedict Etzel)
 1. [General] Added INIT BARO option, INHG/HPA/AUTO - @theomessin (Theodore Messinezis)
+1. [FLIGHTMODEL] Changed rudder and vertical tail parameters for improved crosswind takeoff and landing capability - @donstim - (donbikes#4084)
+1. [FLIGHTMODEL]  Changed full landing flap deflection from 35 degrees to 40 degrees - @donstim - (donbikes#4084)
```
Comment (MisterChocker) [inline .github/CHANGELOG.md:142]:
```suggestion
1. [FLIGHTMODEL] Changed full landing flap deflection from 35 degrees to 40 degrees - @donstim - (donbikes#4084)
```

**Review by MisterChocker — CHANGES_REQUESTED** [review]

**Comment by Lazarus320** [comment]
Category: QA Tester
Name: Lazarus#4286
Date of testing: Current date 8/12/2020
Version of the sim: 1.11.6.0
PR Tested: #2271 
Object of test: Crosswind takeoff and landing capability, flaps degree.  

Testing technique: Wind was set to 38 degrees, weight was 110,000 lbs as well, dry conditions for both takeoff and landing. 
Steps you've made to test the Pull Request:
1. FLaps 40 degree from external view (visually there is no change but on the bottom right corner you can see it changed to 35 to 40) 
image (2)


2. Takeoff: https://youtu.be/Bb2k8qa3qIE  
3. Landing: https://youtu.be/LY7KU5LNatE


Flight notes (If required): Hardware used: Logitech Extreme 3d PRO 

Issues (If Any): N/A
Overall Rating: Good
Conclusions of the testing: Overall everything felt better, I haven't done much crosswind landing or takeoffs due to the addressed behavior but now it feels a lot better and more stable than before, great job!

**Comment by BritishAvgeek** [comment]
Category: QA Tester
Name: Lazarus#4286
Date of testing: Current date 8/12/2020
Version of the sim: 1.11.6.0
PR Tested: #2271
Object of test: Crosswind takeoff and landing capability, flaps degree.

Testing technique: EGCC - Departure, right hand circuit with wind 310/38kts - full crosswind 90 degrees to the runway with 55.68T TOW
Steps you've made to test the Pull Request:
1. Departed EGCC with the wind set to 310/38kts... great behaviour on rotation with the aircraft crabbing into wind during rotation and requiring aileron to counter wing dip
2. Right hand circuit and then made my way to the ILS 23R
3. Final Approach - checked Flaps 40, all good
4. Manual Control of the aircraft from about 5-6nm out to see how it behaved as AP was handling the ILS approach perfectly fine with the 38kt crosswind
5. Landing, correction to counter crab on touch down all good though it feels quite slippery as if I've landed on ice... I had full left rudder to keep it on the centre line at one point



Flight notes (If required): Hardware used: Thrustmaster T-Flight HOTAS X Full Flight Kit

Issues (If Any): N/A
Overall Rating: Good
Conclusions of the testing: The way the aircraft behaved certainly was improved and seemed more natural - with it crabbing into wind during takeoff nicely. Though the controls will need resetting for me to find the new best deadzones/sensitivities with the changes as the rudder input became very harsh randomly... I ran 3 circuits with mixed results with differing settings though the aircraft was definitely better than before having flown into airports in the sim with strong live Wx crosswinds previously including a 30kt into Dublin so was comparing against that too. Great work

**Review by kberg93 — COMMENTED** [review]
Category: QA Tester
Name: Kberg#0666
Date of testing: Current date 12/09/2020
Version of the sim: 1.11.7.0
PR Tested: #2271 
Object of test: Flight model update and full landing flaps to 40 (Crosswind takeoffs and landings)

Testing technique: Take offs and landings in 38kt wind with gusts.

Steps you've made to test the Pull Request: 
1. Take off manually take a round and come back for touch and goes did 6 touch and goes.
2. Then i did 4 takeoffs with landings.

Flight notes (If required): Everything was done flying manual, only autothrottle were used during my test.

Issues (If Any): 

Overall Rating: Good feeling and consistent.

Hardware used: TM Warthog (HOTAS) and MFG Crosswinds v2.

Conclusions of the testing: The rudder feeling was good and i felt it behaved good, the take offs and landings were consistently good without any scetchy or weird behaviour.
Would probably be nice to see what a A320 pilot feels with this PR vs the real deal to get some input maybe?
From my side it felt consistent and good at least. :)

**Pull request merged**